bindata/network/openshift-sdn/sdn-ovs.yaml

```diff
@@ -99,15 +99,26 @@ spec:
         livenessProbe:
           exec:
             command:
-            - /usr/share/openvswitch/scripts/ovs-ctl
-            - status
+            - /bin/bash
+            - -c
+            - |
+              #!/bin/bash
+              /usr/share/openvswitch/scripts/ovs-ctl status > /dev/null &&
+              /usr/bin/ovs-appctl ofproto/list > /dev/null &&
+              /usr/bin/ovs-vsctl show > /dev/null &&
```

Comment:
maybe add comments explaining what each of these tests is testing?

Reply:
done